etc/docker/dev/fts/fts3restconfig

```diff
@@ -19,6 +19,7 @@ Alias=rucio/fts
 MonitoringMessaging=false
 
 OAuth2 = True
+AllowNonManagedTokens = True
```

Comment:
Note: we should also document this in the Rucio documentation

Reply:
Yes will be added to doc.